.travis.yml

```diff
@@ -25,7 +25,7 @@ before_install:
     - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
 
     # Get the latest CMake
-    - wget http://www.cmake.org/files/v3.1/${CMAKE_VERSION}.tar.gz
```

Comment:
I haven't finished debugging this yet, changing the url is not a complete fix.  I just need it to keep running on travis as I iterate... there's some kind of certificate issue I'm digging into